convergence.md

```diff
@@ -1,3 +1,445 @@
+* iojs-vi
+* iojs-ja
+* iojs-tr
+* iojs-cn
+* iojs-ru
+* iojs-fr
+* iojs-hu
+* iojs-pt
+* iojs-uk
+* iojs-ka
+* iojs-fi
+* iojs-fa
+* iojs-ta
+* iojs-it
+* iojs-no
+* iojs-el
+* iojs-ms
+* iojs-ar
+* iojs-he
+
+#### iojs/io.js Branches and Tags:
+
+Branches:
+
+* master
+* mergev010/base
+* mergev010/cp-fixes
+* more-working-groups
+* next
+* old-master
+* only-consensus-seeking
+* roadmap
+* try
+* v0.10
+* v0.12
+* v1.8.0-commit
+* v1.x
+* vupdates
+* working-groups
+
+Tags:
+
+* v1.8.1
+* v1.7.1
+* v1.7.0
+* v1.6.4
+* v1.6.3
+* v1.6.2
+* v1.6.1
+* v1.6.0
+* v1.5.1
+* v1.5.0
+* v1.4.3
+* v1.4.2
+* v1.4.1
+* v1.3.0
+* v1.2.0
+* v1.1.0
+* v1.0.4
+* v1.0.3
+* v1.0.2-release
+* v1.0.2
+* v1.0.1-release
+* v1.0.1
+* v1.0.0-release
+* v1.0.0
+
+Within iojs/io.js, development is currently focused on `master` and `v1.x`.
+
+Current statistics show 11,369 commits (which include some overlap with joyent/node from the fork), 15 branches, 24 releases and 678 contributors.
+
+### Converging the Organizations
+
+1. Ownership of the current `github.org/iojs` organization will transfer to the Node.js Foundation.
+2. The Organization will be renamed from "io.js" to "Node.js Foundation". Note that renaming of the organization will cause the URL for the org to change. Github's infrastructure automatically handles the redirection. For the sake of this document, `github.com/nodejs-foundation` is assumed.
+3. The Owners of the Github Node.js Foundation Organization will include:
+  * The existing joyent node-coreteam members.
+  * The existing iojs TC members.
+  * Additional supporting staff selected by the Foundation Board.
+4. All members of the existing node-coreteam will be added as Collaborators within the organization
+5. A team representing the TSC members will be created (if one does not already exist). The membership of this team will be all current voting members of both the node-coreteam and io.js TC.
+6. Ownership of the following joyent/* repositories will be transferred to the Node.js Foundation organization:
+  * http://github.com/joyent/node
+  * http://github.com/joyent/node-website
+  * http://github.com/joyent/node-advisory-board
+  * http://github.com/joyent/http-parser
+  * http://github.com/joyent/node-tracing ?
+  * http://github.com/joyent/docker-node ?
+7. Once transferred, the `joyent/node` repository will be renamed to `nodejs-foundation/joyent-node`
+8. A new repository will be created called `nodejs-foundation/node`. This repository will become the converged project repository. The existing `nodejs-foundation/joyent-node` and `nodejs-foundation/io.js` repositories would continue in parallel to support each distinct release line.
+
+## Converging the Projects
+
+Once the organization merge has been completed, the `nodejs-foundation/joyent-node` and `nodejs-foundation/io.js` repositories would need to be systematically merged into a single `nodejs-foundation/node` repository. This task will be nontrivial and may be controversial.
```

Comment:
fwiw the commits in question for cherry-picking are: https://github.com/joyent/node/compare/3a9754a...master

(some of which io.js already has.)